CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 
 - changed: Simplify our React Native integration.
+- removed: Delete all rate plugins.
```

Comment:
Why are the rates plugins removed?

Reply:
Since this is a breaking change, we agreed to finally remove these plugins, which are no longer used or maintained.